src/net/server/mod.rs

```diff
@@ -72,14 +72,14 @@ impl MpcHelperServerError {
     }
 }
 
-/// [`From`] implementation for [`MpcServerError::InvalidHeader`]
+/// [`From`] implementation for [`MpcHelperServerError::InvalidHeader`]
```

Comment:
Let's just remove all of these useless comments that say the same thing as the line below. They add no value and as we see here - have a tendency to fall out of sync.